### ISSUE TITLE
File backend should sort points before emitting

### ISSUE THREAD

**Issue by rlgomes** [juttle/juttle#34]

**Comment by dmehra** [comment]
Definitely! Otherwise, given a file with unsorted set of points (ie, disordered by `time` or whatever `-timeField` is set), we will drop most of those points as they get into the juttle flowgraph. I experienced this when setting up data for the tutorial.

Since we'd be sorting in memory, is there a concern for OOM'ing?

**Comment by rlgomes** [comment]
We'd obviously have to set a `-limit` on the number of points to be handled when sorting.
